data/json/items/containers/containers.json

```diff
@@ -147,7 +147,7 @@
     "name": { "str": "canvas bag" },
     "looks_like": "bag_canvas",
     "description": "A small bag made of canvas.  Looks fine to store dried herbs in.",
-    "weight": "5 g",
+    "weight": "20 g",
     "volume": "250 ml",
```

Comment:
I don't think folded bag should be that big

Reply:
![ary](https://github.com/CleverRaven/Cataclysm-DDA/assets/146848158/f37925a3-001e-4aec-b35a-1baa50670b89)

Reply:
assuming canvas density of 1.42e3 (from Google) volume should be about 0.02 / 1.42e3  = 0.0000140845 q m or about 14 ml

Reply:
should I update it, because I've been told on discord that volumes less than 250ml not handled correctly

Reply:
Volume of pockets, that should contain water, less than 250 ml not handled correctly, to be specific - which is not the case here
Canvas density in the game is 1.6 (don't know the reason, google says 1.4, but the difference isn't huge), so i assume 14 ml folded should be fine
```suggestion
    "volume": "14 ml",
```